R/get_population.R

```diff
@@ -335,7 +335,7 @@ get_population <- function(kingco = T,
                         sql_col = c('race_aian', 'race_as', 'race_blk', 'race_hisp', NA, 'race_nhpi', 'race_wht'))
   stopifnot(nrow(ref.table) == 7)
   ref.table = merge(ref.table, colnames, all.x = T, by = 'short')
-  if(!all(races == 'All')) races = ref.table[ short %in% races, value]
```

Comment:
I'm not sure why this change would make a difference in addressing the problem identified by Carolina. I would have coded it the previous way. Regardless, it seems logical (though unconventional ... why have %in% single_string?

Reply:
Yeah -- this change isn't really required. I just was trying to reduce instances of `==` for vectors with length >1